Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControlTypes.cs

```diff
@@ -107,4 +107,19 @@ internal enum CardinalAxisType
         Y,
         Z
     }
+
+    /// <summary>
+    /// Scale mode that is used for scaling behavior of bounds control.
+    /// </summary>
+    public enum HandleScaleMode
+    {
+        /// <summary>
+        /// Control will be scaled uniformly.
+        /// </summary>
+        Uniform,
+        /// <summary>
+        /// Precise scaling scales non uniformly according to movement in 3d space.
+        /// </summary>
+        Precise
```

Comment:
I am  not that non-uniform is the same as precise. Should we just call it non-uniform? Precise could be a separate mode...

Reply:
i wasn't sure about the naming - i can change it to non-uniform. by saying precise is a different mode are you referring to the precision tools from finn?

Reply:
changed to non uniform